src/components/ParameterChangeForm.tsx

```diff
@@ -16,7 +16,7 @@ import { updateSearchString } from "../utils/updateSearchString";
 import { EditableTable, RowValue } from "./EditableTable";
 import { ParamsTypeSelector } from "./ParamsTypeSelector";
 import { BeansPerUnit } from "../types/swingset";
-import type { FormValue, ParameterChangeTypeOption } from "../types/form";
+import type { FormValue, ParameterChangeTypeDescriptor } from "../types/form";
```

Comment:
I don't see any product requirements that motivate this rename. I suggest leaving it out.

If "descriptor" has precedent in cosmos tech, please cite it.